bin/pfioh

```diff
@@ -21,11 +21,42 @@ import  pudb
 # and related dependencies.
 sys.path.insert(1, os.path.join(os.path.dirname(__file__), '../pfioh'))
 try:
-    import  pfioh
+    import pfioh
+    import mount_dir
+    import swift_store
 except:
     sys.path.insert(1, os.path.join(get_python_lib(True, True), 'site-packages'))
     sys.path.insert(1, os.path.join(get_python_lib(True, True), 'site-packages/pfioh'))
-    import  pfioh
+    sys.path.insert(1, os.path.join(get_python_lib(True, True), 'site-packages/mount_dir'))
```

Comment:
This is only relevant if we are building mount_dir and swift_store as separate packages, which I don't think you are.